e2e/pulsar_test/pulsar_test.go

```diff
@@ -33,12 +33,15 @@ import (
 
 // Pulsar configuration. Must be manually reconciled with changes to the test setup or Armada.
 const pulsarUrl = "pulsar://localhost:6650"
```

Comment:
Why don't we set this const with the others? I think this was a sanity-fail by the lint autofix...